src/ensembl/src/content/app/entity-viewer/shared/components/default-transcript-label/DefaultTranscriptLabel.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import { Transcript } from 'src/content/app/entity-viewer/types/transcript';
+import QuestionButton from 'src/shared/components/question-button/QuestionButton';
+
+import styles from './DefaultTranscriptLabel.scss';
+
+type DefaultTranscriptLabelProps = {
+  transcript: Transcript;
+};
+
+const transcriptLabel = {
+  selected: {
+    label: 'Selected',
+    helpText:
+      'The selected transcript is a default single transcript per protein coding gene that is representative of biology, well-supported, expressed and highly conserved'
+  }
+};
+
+export const DefaultTranscriptLabel = (props: DefaultTranscriptLabelProps) => {
+  const CanonicalType = props.transcript && 'selected'; // TODO Change this to props.transcript.mane/plus etc when available
```

Comment:
eslint was complaining about props not being used. So just added this props.transcript here. Will have to change accordingly when we get the actual data from Thoas.

Reply:
Ok, this looks good; I just have several minor nitpicking remarks:

- We don't usually use pascal case (as in `CanonicalType`) for variable names
- The word `canonical` refers to only a single transcript. I.e. one and only one transcript among all gene transcripts will be designated as canonical. Therefore, the name "canonical type" is somewhat misleading. I think what you are after is `transcriptSupportLevel`

Reply:
transcriptSupportLevel or transcriptSupportLabel?

Reply:
I was thinking on level.

On the other hand, googling now, I see that transcript support level is what gave the acronym TSL (TSL-1, TSL-2, etc). Ensembl seems to be struggling with proposing a generic term for all of that (see https://www.ensembl.org/info/genome/genebuild/transcript_quality_tags.html). The url contains a good suggestion — `transcriptQuality`. In the [docs](https://github.com/Ensembl/ensembl-e2020-datafiles/blob/master/docs/genes_and_transcripts.md) to the script that builds data files for genome browser we are calling it "transcript designation". I think Dan used the word "prestige"; not sure whether he is still using it.

Reply:
transcriptQuality sounds simple and easy to understand, but @ens-st3 do you have any suggestions?